cpp/src/slice2rb/RubyUtil.cpp

```diff
@@ -43,7 +43,7 @@ namespace
         Slice::validateMetadata(unit, "ruby", std::move(knownMetadata));
     }
 
-    string getEscapedParamName(const OperationPtr& p, const string& name)
+    string getEscapedRubyParamName(const OperationPtr& p, const string& name)
```

Comment:
Can't use the Util function because of the special Slice::Ruby::getMappedName logic.